templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -472,6 +480,40 @@ <h2>
     </section>
   </div>
 
+  <hr class="p-rule is-fixed-width" />
+
+  <div class="u-fixed-width">
+    <section class="p-section">
+      <div class="row--50-50">
+        <div class="col">
+          <h2>See Ubuntu Pro in action</h2>
+        </div>
+        <div class="col">
+          <p class="p-section--shallow u-no-padding--bottom">
+            Get long term support and automated vulnerability management for all your devices, and streamline your patchway to compliance with Ubuntu Pro, the proven way to meet major compliance requirements.
+          </p>
+          <hr class="p-rule--muted" />
+          <p>
+            Airlock Digital meets stricter security and compliance requirements with UBuntu Pro – with up to 40% cost savings and performance increases.
+          </p>
+          <p>
+            <a href="/case-study/airlock-digital-ubuntu-pro-azure-case-study">Download the case study</a>
+          </p>
+          <hr class="p-rule--muted" />
+          <p>Lucid Software meet FedRAMP compliance for government contracts through Ubuntu Pro</p>
+          <p>
+            <a href="/case-study/lucid-aws-fedramp-compliance-case-study">Download the case study</a>
+          </p>
+          <hr class="p-rule--muted" />
+          <p>LaunchDarkly becomes the first FedRAMP-authorised feature management platform thanks to Ubuntu Pro</p>
+          <p>
+            <a href="https://ubuntu.com/engage/LaunchDarkly-FedRAMP-case-study">Download the case study</a>
+          </p>
```

Comment:
When a person is using screen-reader all the "download case study" links at same place will be read in-differently if we don't add aria-labels explaining a bit about what is being downloaded by clicking that button.
Look in repo and you would find examples for Aria-labels.